src/components/Main/OrderDeteils/index.js

```diff
@@ -0,0 +1,233 @@
+// import { borderBottom } from "@mui/system";
+import styled from "styled-components";
+import Image from "../Image";
+import { useState } from "react";
+import { Formik, Form, Field, ErrorMessage } from "formik";
+import { validationRules } from "../../../Utils/Validation/validations";
+import { SignupSchema } from "../../../Utils/Validation/validationShema";
+import { createOrder } from "../../../Utils/Apis";
+import { Button, Stack } from "@mui/material";
+import Loader from "../Loader";
+
+const style = {
+  backgroundColor: "Black",
+  display: "flex",
+  justifyContent: "center",
+  width: "160px",
+  height: "50px",
+};
+
+const OrderDetails = ({ quantities, totalPrice, clearBurger, cancel }) => {
+  const [loading, setLoading] = useState(false);
+  const [checked, setChecked] = useState(false);
+  const [orderCreateStatus, setOrderCreateStatus] = useState(false);
+  const [sendOrderStatus, setSendOrderStatus] = useState(false);
+
+  const burgerComponents = Object.entries(quantities).filter((el) => el[1]);
+
+  const orderWindowClesed = () => {
+    cancel();
+    setSendOrderStatus(false);
+    setOrderCreateStatus(false);
+  };
+  const handleOrder = async (data) => {
```

Comment:
I was able to place an empty burger order, it should be fixed

Reply:
fixed